devicetypes/smartthings/zooz-4-in-1-sensor.src/zooz-4-in-1-sensor.groovy

```diff
@@ -24,6 +24,7 @@ metadata {
 		capability "Tamper Alert"
 
 		fingerprint mfr: "027A", prod: "2021", model: "2101", deviceJoinName: "Zooz 4-in-1 sensor"
+        fingerprint mfr: "0109", prod: "2021", model: "2101", deviceJoinName: "ZP3111US 4-in-1 Motion"
```

Comment:
add my company's fingerprint

Reply:
Please make sure the indentation is the same as other fingerprints

Reply:
I had  indentation the same as other fingerprints
Thank you